ui/src/app/pages/homepage/quick-tour-modal.tsx

```diff
@@ -360,8 +354,18 @@ export class QuickTourReact extends React.Component<QuickTourReactProps, QuickTo
           <div style={{width: '100%', paddingTop: '5%'}}>
             <div style={styles.divider}/>
           </div>
-          <div style={styles.panel}>
-            <div style={{width: '75%'}}>
+          <div style={{
+            flex: 1,
+            minHeight: 0,
+            marginTop: '5%',
```

Comment:
Can we extract the style for better readability?

Reply:
On this one, I would be ok extracting the flex, minHeight and marginTop, but I do not want to do that with the grid.

The CSS grid template has meaning that effects its child elements. When reading through the code it would be difficult to have to bounce back and forth to figure out the grid layout - I think it would make it more difficult to maintain.

Reply:
factored out non-grid styling